src/main/java/com/ultimatelogger/multiplatform/tag/dataprovider/TagDataConverter.kt

```diff
@@ -2,8 +2,12 @@ package com.ultimatelogger.multiplatform.tag.dataprovider
 
 internal class TagDataConverter {
 
+    companion object {
+        private const val FILE_NAME_MISSING = "UnknownFile"
+    }
+
     fun fromStackTraceElement(stackTraceElement: StackTraceElement) =
-            TagData(stackTraceElement.fileName,
+            TagData(stackTraceElement.fileName ?: FILE_NAME_MISSING,
                     stackTraceElement.className,
```

Comment:
Do you know if it's possible for the other elements to be null to?

Reply:
![image](https://user-images.githubusercontent.com/24584689/72669063-bef6b500-3a2d-11ea-8398-56bc77eb3d06.png)